data_conversion/data_conversion_merge/Data_conversion_merge_CSV.py

```diff
@@ -1,61 +1,82 @@
-'''
+"""
 Combines csv files for different types of capped repeat arrays (NRC homopolymer
 arrays and NRXC heteropolymer arrays with point substitions (here T4V in
 consensus ankyrin repeat arrays, creates numpy files for each melt, and 
 generates a melt list and a construct list.
 
 Most recent revision: 05/21/2020, Doug Barrick
 
-'''
+"""
 
 import numpy as np
 import pandas as pd
 import json
 
-path = '/Users/dougbarrick/OneDrive - Johns Hopkins/Manuscripts/Ising_program/\
-Scripts/scripts_mutations/T4V_two_mi_values_2020_05_10/'
+path = "/Users/dougbarrick/OneDrive - Johns Hopkins/Manuscripts/Ising_program/\
+Scripts/scripts_mutations/T4V_two_mi_values_2020_05_10/"
 
-proj_name = 'T4V_NRC_2mi'
+proj_name = "T4V_NRC_2mi"
 
 den_nsig_const_melt = []
-constructs = []         # List of constructs used to build partition functions in next script
-melts = []              # List of melts to be used in fitting script
+constructs = (
+    []
+)  # List of constructs used to build partition functions in next script
+melts = []  # List of melts to be used in fitting script
 
-T4V_input_df = pd.read_csv('{}T4Vdata_not_normalized.csv'.format(path),names=['denat','signal','construct_melt','dataset'])
-NRC_input_df = pd.read_csv('{}NRC_data_dnmn.csv'.format(path),names=['denat','signal','construct_melt','dataset'])
-maxT4Vmelt = T4V_input_df['dataset'].max()  # Finds the maximum number of melts in the first df
-NRC_input_df['dataset'] = NRC_input_df['dataset'] + maxT4Vmelt # Adds the max number to the melt numbers in second df
-combined_input_df = pd.concat([T4V_input_df, NRC_input_df],names=['denat','signal','construct_melt','dataset'])
-combined_input_df.to_csv('{}T4V_NRC_dnmn.csv'.format(path), index=False, header=False)
+T4V_input_df = pd.read_csv(
+    "{}T4Vdata_not_normalized.csv".format(path),
```

Comment:
You can see how black helps readability in these cases where you have many arguments and long line length